docs/make.jl

```diff
@@ -19,8 +16,4 @@ makedocs(
           ]
 )
 
-deploydocs(deps = nothing, make = nothing,
```

Comment:
Other keywords used they value by default, so we can remove them.
I'm still trying to find why, after deployment, links to source don't work...